src/containers/LanguageSwitcher.tsx

```diff
@@ -0,0 +1,32 @@
+import { ChangeEvent } from "react";
+import Form from "react-bootstrap/Form";
+import { useTranslation } from "react-i18next";
+
+const lngs: Record<string, string> = {
```

Comment:
```suggestion
const languages: Record<string, string> = {
```
why not to use normal name?